packages/agents/sdk/src/sdkShared.ts

```diff
@@ -92,6 +92,14 @@ export class NxtpSdkShared {
     return undefined;
   }
 
+  async getAssetsData(): Promise<any> {
```

Comment:
Can we add a type for this? Since it's a SDK response it's public and it shouldn't be `any`.

Reply:
added it!